source/Hud.hx

```diff
@@ -0,0 +1,85 @@
+package;
+
+import flixel.text.FlxText;
+import flixel.FlxG;
+import flixel.util.FlxColor;
+import flixel.system.debug.Window;
+import flixel.FlxSprite;
+import flixel.group.FlxGroup.FlxTypedGroup;
+
+class Hud extends FlxTypedGroup<FlxSprite> {
+
+    var background:FlxSprite;
+    var livesSprite:FlxSprite;
+    var livesCounter:FlxText;
+    var scoreSprite:FlxSprite;
+    var scoreCounter:FlxText;
+    var abilitySprite:FlxSprite;
+    var abilityBar:FlxSprite;
+    var lives:Int;
```

Comment:
In Sam's branch, he ended up using the `Player` class's built-in `health` property (inherited from `FlxObject`) to track the player's remaining 'life'. 

You should be able to rely on it for tracking this number instead, though you'll need to communicate changes to that value from within player somehow.